### DISCUSSION TITLE
Ikon GSR1 from NYSEG

### DISCUSSION THREAD

**Discussion by rgusick** [bemasher/rtlamr#300] [General]
[NYSEG](https://www.nyseg.com/smartenergy/innovation/smartmeters/installation) just installed an **Ikon GSR1** (https://fccid.io/SK9G5R1) smart meter at my house. I don't see it listed as a model that has been  tested with rtlamr in the [meters.csv](https://github.com/bemasher/rtlamr/blob/master/meters.csv) file. Is this because it is known not to be compatible, or that this model just hasn't been tested yet?

![image](https://github.com/bemasher/rtlamr/assets/116862/28c94ab3-06fa-477d-b813-f21ecce1b2c9)

**Comment by dcorwin8222** [comment]
Any luck with this meter? I have the same meter from NYSEG.
